UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json

```diff
@@ -0,0 +1,1799 @@
+            }
+          ],
+          "text": "Middle Initial",
+          "type": "string"
+        },
+        {
+          "linkId": "PND.2",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"BasicPractitionerInfoPrepopulation\".NPI"
+              }
+            }
+          ],
+          "text": "NPI",
+          "type": "string"
+        }
+      ]
+    },
+    {
+      "linkId": "ENC",
+      "text": "Encounter",
+      "type": "group",
+      "item": [
+        {
+          "linkId": "ENC.4",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"VentilatorsPrepopulation\".Today"
+              }
+            }
+          ],
+          "text": "Date of encounter (MM/DD/YYYY):",
+          "type": "date",
+          "required": true
+        },
+        {
+          "linkId": "UROENC.1",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"UrologicalSuppliesPrepopulation\".AnswerYes"
+              }
+            },
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "code": "radio-button",
+                    "system": "http://hl7.org/fhir/questionnaire-item-control"
+                  }
+                ]
+              }
+            }
+          ],
+          "text": "Is this an evaluation of the patient’s need for urological supplies?",
+          "required": true,
+          "type": "choice",
+          "answerValueSet": "#yesno",
+          "item": [
+            {
+              "linkId": "UROENC.1a",
+              "text": "If No, purpose of the encounter",
+              "type": "text",
+              "require": true,
+              "enableWhen": [
+                {
+                  "question": "UROENC.1",
+                  "operator": "=",
+                  "answerCoding": {
+                    "code": "N"
+                  }
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "DIAG",
+      "text": "Diagnoses",
+      "type": "group",
+      "item": [
+        {
+          "linkId": "DIAG.1",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"VentilatorsPrepopulation\".CurrentDiagnoses"
```

Comment:
On the form, I don't see this is populated. 
![Screen Shot 2020-05-06 at 9 39 46 AM](https://user-images.githubusercontent.com/58572522/81183792-96afa980-8f7d-11ea-92fe-3cfc44f3ce81.png)

Reply:
Copy/Past error. Fixed